packages/driver/src/cy/mouse.ts

```diff
@@ -707,6 +707,8 @@ const create = (state, keyboard, focused, Cypress) => {
   return mouse
 }
 
+export interface Mouse extends ReturnType<typeof create> {}
```

Comment:
nit: I personally like when exports are at the end of files so it's easier to find. Thoughts?

Reply:
I also thought about it at first but I placed it there because it would be better to place `export` functions in one place. 

But it seems that your idea is better. I changed it.